assets/js/search.js

```diff
@@ -122,21 +122,37 @@ function titleExtractor (document) {
 
   if (type === 'function' || type === 'callback' || type === 'type') {
     var modFun = title.replace(/\/\d+/, '')
-    var modOrFun = modFun.replace('.', ' ')
+    var modOrFun = modFun.replace(/\./g, ' ')
```

Comment:
I assume this change is necessary as otherwise it replaces only the first one?

Reply:
Correct. This is expanding the name to make it searchable, so `Mod.Mod2.Fun` was only getting split into `Mod Mod2.Fun` so you couldn't search for `Mod2` or `Fun` independently.